drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -101,9 +101,9 @@ def hud_id
     end
 
     def enrollments(**args)
-      return Hmis::Hud::Enrollment.none unless current_user.can_view_enrollment_details_for?(object)
+      return unless current_user.can_view_enrollment_details_for?(object)
```

Comment:
So this is checking that the user has the correct access right?  I wonder if we should have a note here that says something to the effect of "skipping permission checks below for performance, ensure the user can access this enrollment here and won't re-check"